sdk/python/feast/infra/online_stores/sqlite.py

```diff
@@ -114,7 +114,7 @@ def online_write_batch(
 
                     conn.execute(
                         f"""INSERT OR IGNORE INTO {_table_id(project, table)}
-                            (entity_key, feature_name, value, event_ts, created_ts)
```

Comment:
intentional? revert?

Reply:
yes it's intentionally happen, while formatting the code